Sources/MarathonCore/Marathon.swift

```diff
@@ -40,8 +40,9 @@ public final class Marathon {
             let rootFolder = try fileSystem.createFolderIfNeeded(at: folderPath)
             let packageFolder = try rootFolder.createSubfolderIfNeeded(withName: "Packages")
             let scriptFolder = try rootFolder.createSubfolderIfNeeded(withName: "Scripts")
+            let autocompletionsFolder = try rootFolder.createSubfolderIfNeeded(withName: "Autocompletions")
```

Comment:
Let's just do that for now in this PR; and then we can continue working on this in subsequent ones 😄 Just update that folder name and I'll merge this in 👍